### PR TITLE
Q3 besu update

### PR DESCRIPTION
**Pull request by jflo**

### CODE REVIEW

**Comment by tkuhrt** [comment]
@jflo : can you please migrate this PR to our new repo: https://github.com/LF-Decentralized-Trust/governance under the `tac\project-updates` folder?

**Comment by tkuhrt** [comment]
> @jflo : can you please migrate this PR to our new repo: https://github.com/LF-Decentralized-Trust/governance under the `tac\project-updates` folder?

Never mind. It appears this is the Q3 report. This is the right spot for that. For Q4, we will move to the new repo. My apologies.

File: project-reports/2024/2024-Q3-Hyperledger-Besu.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Current Plans
+
+While the Prague and Osaka upgrades remain the projects primary focus, in Q4 we continue to chip away at technical debt and modularity work. Feature sunset has been announced [here](https://www.lfdecentralizedtrust.org/blog/sunsetting-tessera-and-simplifying-hyperledger-besu). A big focus of the maintainers will be on performance and stability in Q4, targeting a much higher gas throughput number in 2025 for Besus execution (road to Gigagas). 
+
+Non-protocol feature work is largely paused in this quarter besides Bonsai Archive and work on the plug-in API alongside new documentation to attract plug-in devs and L2s to use the system.
+
+# Maintainer Diversity
+
+Luis Pinto was added as a maintainer this quarter, and multiple, full-time contributors have joined the team - we expect them to earn maintainer status later in Q4.  Part-time contributions continue and are increasing.
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Besu.md:44]: What organization is Luis and the other full-time contributors working towards maintainer status from?

Comment (jflo) [reply project-reports/2024/2024-Q3-Hyperledger-Besu.md:44]: Consensys

**Review by swcurran — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Besu.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Besu
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+The Besu project is thriving at the moment. There's growing interest in both public blockchain networks and private chains, with several Layer 2 solutions considering Besu for their infrastructure. Publicity around the work on public/private feature parity and new contributions is also increasing. Community activity on Discord has risen, contributions are up, and existing contributors and maintainers are collaborating more effectively than before. Overall, the project's health is robust. The #besu-enterprise channel has been effective in directing new interest in private networks to the right support teams. Discord support and engagement with maintainers and documentation remain strong.
```
Comment (non-fungible-nelson) [inline project-reports/2024/2024-Q3-Hyperledger-Besu.md:10]: Have we seen any inflow regarding the sunset plan? And have the maintainers discussed a plan with LFDT and amongst contributors to address any necessary issues?

**Review by non-fungible-nelson — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**